typings/dataWrangler.d.ts

```diff
@@ -0,0 +1,260 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Event, Uri } from 'vscode';
+import type { Kernel } from '@jupyterlab/services/lib/kernel';
+import type { Session } from '@jupyterlab/services';
+
+/**
+ * Data Wrangler extension exports.
+ */
+export interface IDataWranglerExtensionAPI {
+    /**
+     * Promise indicating whether all parts of the extension have completed loading or not.
+     * @type {Promise<void>}
+     * @memberof IDataWranglerExtensionAPI
+     */
+    ready: Promise<void>;
+
+    /**
+     * Launches the Data Wrangler using the given variable name.
+     */
+    launchDataWranglerUsingVariable: (
```

Comment:
If this PR is to allow launching the DW variable explorer, then I think we can just add an API extension in our extension that allows any other extension to create entry points for launching variable viewer for any given variable.

That way we don't tightly couple DW with the Jupyter extension.

Reply:
discussed offline, this is for the output renderer